.travis.yml

```diff
@@ -56,24 +59,30 @@ env:
   - secure: KX7DwKRD85S7NgspxevgbulTtV+jHQIiM6NBus2/Ur/P0RMdpt0EQQ2wDq79qGN70bvvkw901N7EjSYd+GWCAM7StXtaxnLRrrZ3XI1gX7KMk8E3QzPf0zualLDs7cuQmL6l6WiElUAEqumLc7WGpLZZLdSPzNqFSg+CBKCmTI8=
 
 git:
-  depth: 60
+  depth: 50
```

Comment:
The depth increase is so we can infer the most recent tag for setting a version?

Reply:
Yeah.. I've found that it doesn't seem to add that much to the checkout time or `.git` size, since the really big stuff is probably hundreds of commits old at this point. And I've found that my own typical work on a dev branch can approach 50 commits prior to a PR unless I squash it, and if it goes beyond that, it fails to deploy the conda and docker packages because it can't figure out the version tag.